vue/src/core/components/input-widget/InputWidget.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <ValidationProvider :name="`${attribute}-${uuid}`" :rules="!rules ? newRules : rules"
+                      :customMessages="model.defaultRules" v-slot="v" tag="div" :vid="vid">
```

Comment:
We should call `:customMessages="model.getMessages()"` which will merge model's rule messages with `defaultMessages`.